test/functional/http/get/accounts.multisignatures.js

```diff
@@ -14,9 +14,8 @@
 'use strict';
 
 var test = require('../../functional.js');
-var _ = test._;
+
```

Comment:
When you're removing a line, it's best to remove it entirely rather than leaving lots of extra blank space. (Also applies in other locations.)